src/main/java/app/JeM.java

```diff
@@ -0,0 +1,169 @@
+package app;
+
+import taskmanager.Storage;
+import exceptions.InvalidCommandException;
+import tasks.Task;
+import tasks.Todo;
+import tasks.Deadline;
+import tasks.Event;
+
+import java.util.Scanner;
+
+public class JeM {
```

Comment:
You can consider using Jem instead to follow the PascalCase class naming format.